pyproject.toml

```diff
@@ -5,16 +5,18 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "bqt"
 version = "1.4.2" # don't forget to update the version in blender_manifest.toml
-description = "Files to help bootstrap PySide2 with an event loop within Blender."
+description = "Files to help bootstrap PySide6 with an event loop within Blender."
 readme = "README.md"
 requires-python = ">=3.11"
 classifiers = [
-    "Development Status :: 4 - Beta",
+    "Development Status :: 5 - Production/Stable",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3.7",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)",
```

Comment:
This was omitted on purpose. It is no longer supported by newer build chains see https://peps.python.org/pep-0639/

License are now specified like this
```
license = "MPL-2.0"
license-files = ["LICEN[CS]E*"]
```

Note that the `license` entry needs to be a valid [license ref](https://packaging.python.org/en/latest/specifications/license-expression/#specification) we can not use the full name (afaik).